README.md

```diff
@@ -81,6 +81,7 @@ SNOWFLAKE_DB_PASSWORD=
 SNOWFLAKE_DB_DATABASE=
 SNOWFLAKE_DB_SCHEMA=
 SNOWFLAKE_DB_WAREHOUSE=
+STORAGE_API_URL=
```

Comment:
Proč to najednou potřebuje storage api url?

Reply:
Tj. jak to souvisí se změnou loggeru a ssh tunelu?

Reply:
je to pryč - bylo to tím že jsem musel updatovat i storage api, ale vyřešil jsem to jinak a je to zpátky na verzi ^4.14